api/model/src/main/java/org/projectnessie/api/v2/http/HttpTreeApi.java

```diff
@@ -110,6 +115,36 @@ public interface HttpTreeApi extends TreeApi {
   @JsonView(Views.V2.class)
   ReferencesResponse getAllReferences(@BeanParam @jakarta.ws.rs.BeanParam ReferencesParams params);
 
+  @Override
+  @POST
+  @jakarta.ws.rs.POST
+  @Produces(MediaType.APPLICATION_JSON)
+  @jakarta.ws.rs.Produces(jakarta.ws.rs.core.MediaType.APPLICATION_JSON)
+  @Path(".all")
+  @jakarta.ws.rs.Path(".all")
```

Comment:
`POST /api/v2/trees/.all` to _fetch_ references sounds counter-intuitive to me (based on REST principles).

Reply:
We have to use POST/PUT in this case, because CEL filters still contain problematic characters.

Reply:
but they are not in URI paths :thinking:

Reply:
> but they are not in URI paths 🤔

:facepalm: